spec/lib/update_missing_business_entities_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rspec'
+
+describe UpdateMissingBusinessEntities do
+
+  subject(:transform) { described_class.new }
+
+  let(:business_entity) { create :business_entity }
+  let(:shared_params) { { office: business_entity.office, jurisdiction: business_entity.jurisdiction } }
+  let(:shared_params_with_business_entity) { { business_entity: business_entity, office: business_entity.office, jurisdiction: business_entity.jurisdiction } }
+  let!(:application1) { create(:application_full_remission, :processed_state, shared_params) }
+  let!(:application2) { create(:application_full_remission, :processed_state, shared_params_with_business_entity) }
+  let!(:application3) { create(:application_full_remission, :processed_state, shared_params) }
+
+  describe '#up!' do
+    it 'works in sequence' do
+      expect(affected?).to eql(2)
```

Comment:
affected could be UpdateMissingBusinessEntities.affected_records (if you made it a class method)